### ISSUE TITLE
putGeoipDatabase always sets body to undefined

### ISSUE THREAD

**Issue by ElenaStoeva** [elastic/elasticsearch-js#2377]
### 🐛 Bug report

Requests for creating GeoIP database using the JS client fail with error "request body is required". This is because the body is set to undefined:

![Image](https://github.com/user-attachments/assets/738bcacc-a37c-45ea-a6ed-6f3fed66a82a)

### To reproduce

In Kibana, we tried running the following request and it fails with 400 Bad request error "request body is required":
```
clusterClient.asCurrentUser.ingest.putGeoipDatabase({
  id: '<database-id>',
  body: {
    name: '<database-name>',
    maxmind: {
      account_id: '<maxmind-id>',
    },
  },
});
```

### Expected behavior

The body should not be set to undefined.

### Node.js version

20.x

### @elastic/elasticsearch version

8.16

### Operating system

macOS

**Comment by ElenaStoeva** [comment]
Hi @JoshMock, thanks for responding to the issue. I see the bug is still in the [v8.15.0 tag](https://github.com/elastic/elasticsearch-js/blob/599d7e6e07023b0ce7fabec60f41a1497c8965ec/src/api/api/ingest.ts#L260) which I think is the version used by Kibana. Do you know when this tag would be updated?

**Comment by JoshMock** [comment]
Oops, that's my bad! I thought it had been released but it looks like that fix merged after 8.15.0 was published.

I'm currently working on a bug fix in `@elastic/transport` and will publish 8.15.1 as soon as that is done.

**Comment by JoshMock** [comment]
8.15.1 has now been published. Take a look and see if it fixes your problem.
